doc/source/GettingStarted/ConfiguringGafferForThirdPartyTools/index.md

```diff
@@ -9,8 +9,11 @@ Gaffer is compatible with the following commercial and open-source third-party t
 
 Gaffer comes with Appleseed, so it will require no additional configuration. For the rest of the tools in this list, you will need to set some additional environment variables.
 
+> Tip :
+> If you do not use Appleseed in production, you can hide it's nodes from the node menu by setting the `GAFFERAPPLESEED_HIDE` environment variable to `1`. Even when set, Appleseed will still be available for OSL shader previews.
```

Comment:
it's -> its

Reply:
Good spot, thanks, fixed in https://github.com/GafferHQ/gaffer/commit/4803965593c4e9fcc456fbe3f8bd9497eb1f6f7c.